frontend/e2e/app.spec.ts

```
@@ -115,9 +128,14 @@ test("handles invalid URL error", async ({page}) => {
   const dialog = await dialogPromise;
   expect(dialog.message()).toContain("Could not perform request");
   await dialog.accept();
+
+  expect(errors).toEqual([]); // fails if any console.error occurred
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)